src/components/textarea/textarea.component.tsx

```diff
@@ -129,6 +129,8 @@ export interface TextareaProps
   borderRadius?: BorderRadiusType | BorderRadiusType[];
   /** Hides the borders for the component. Please note that validation and focus styling will still be applied */
   hideBorders?: boolean;
+  /** Specify the minimum height */
+  minHeight?: number;
```

Comment:
suggestion: for consistency we should probably keep the type as string

maxWidth is a string and we shouldn't really limit the user to using px when they might want relative heights